charts/stable/nginx-ingress/defaults.yaml

```diff
@@ -1,4 +1,4 @@
 component: nginx-ingress
 gitUrl: https://github.com/helm/charts/tree/master/stable/nginx-ingress
 namespace: nginx
-version: 1.39.1
+version: 1.41.3
```

Comment:
It might be best to leave the nginx version update to this PR https://github.com/jenkins-x/jxr-versions/pull/1791 which will also include an update to `jx gitops upgrade` which also changes the helmfile repository location to the new nginx location.

Reply:
nginx-ingress to ingress-nginx is not a seamless switch. Considering that, shall we leave the version update in?

Reply:
The switch is being worked on in the other PR linked which we will likely need before beta in the next week or so.

Reply:
I did see that. All I am saying is, this version update is unrelated to the switch

Reply:
letting this go :slightly_smiling_face: